growthbook/growthbook
# Demo datasource project global banner

### Features and Changes

This banner will warn users when they're in the demo datasource project so that they don't accidentally create resources under it. 

Clicking on the text will show a tooltip. Clicking on the text again or anywhere else on the screen will hide the tooltip.

It's gated behind a feature flag and will not be shown in demo datasource projects on prod without the feature being enabled.

### Dependencies

n/a

### Testing

1. Create a project with the demo ID. You can create any project and change the ID to the demo ID. See https://github.com/growthbook/growthbook/pull/1477 for the function you can run locally to get the demo ID for your organization.
2. Load any page with the demo project selected
3. See the banner


### Screenshots

When the project is the demo datasource project:

<img width="1788" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/9bcca49c-f7f8-4dad-a398-2de6a4ce2fcb">

<img width="541" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/9b91238f-f49b-4028-abef-38e4eafa9d1d">

And on longer pages (org settings) it's sticky.

<img width="1302" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/1d18dfd2-9940-4d96-88c1-dd76f932c9fc">



No project or a different project:

<img width="1668" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/6b0fd5c1-152e-4355-b594-46a8f1fa3acb">
